*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\ANG - Angola.txt	2019-01-07 18:39:05.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\ANG - Angola.txt	2019-01-07 18:43:55.000000000 -0800
***************
*** 82,93 ****
--- 82,94 ----
  add_opinion_modifier = { target = RME modifier = positive_199 }
  add_opinion_modifier = { target = SAF modifier = positive_105 }
  add_opinion_modifier = { target = UTA modifier = positive_15 }
  
  
  add_ideas = {
+ 	RME_former_dominion
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
