*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\ANG - Angola.txt	2018-12-31 20:21:00.000000000 -0800
--- D:\My Projects\RomanHoI4\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\ANG - Angola.txt	2018-12-31 21:51:37.000000000 -0800
***************
*** 167,178 ****
--- 167,179 ----
  add_opinion_modifier = { target = TUN modifier = positive_0 }
  add_opinion_modifier = { target = UTA modifier = positive_15 }
  add_opinion_modifier = { target = YUG modifier = positive_0 }
  
  
  add_ideas = {
+ 	RME_former_dominion
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
