*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\AOT - Aoteorea.txt	2019-01-10 18:41:55.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\AOT - Aoteorea.txt	2019-01-10 16:12:23.000000000 -0800
***************
*** 76,97 ****
  add_opinion_modifier = { target = SHN modifier = negative_10 }
  add_opinion_modifier = { target = SOK modifier = positive_5 }
  add_opinion_modifier = { target = UTA modifier = negative_15 }
  
  
  add_ideas = {
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
  set_war_support = 0.55
  create_country_leader = {
!     name = "Nikolaos Nider"
!     desc = "POLITICS_NIKOLAOS_NIDER_DESC"
      picture = "GFX_GRE_Panagis_Tsaldaris"
      expire = "1965.1.1"
      ideology = socialism
      traits = {
      }
  }
--- 76,98 ----
  add_opinion_modifier = { target = SHN modifier = negative_10 }
  add_opinion_modifier = { target = SOK modifier = positive_5 }
  add_opinion_modifier = { target = UTA modifier = negative_15 }
  
  
  add_ideas = {
+ 	RME_former_dominion
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
  set_war_support = 0.55
  create_country_leader = {
!     name = "Theodoros Dousmanis"
!     desc = "POLITICS_THEODOROS_DOUSMANIS_DESC"
      picture = "GFX_GRE_Panagis_Tsaldaris"
      expire = "1965.1.1"
      ideology = socialism
      traits = {
      }
  }
