*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\AQI - Aquitaine.txt	2018-12-31 22:05:12.000000000 -0800
--- D:\My Projects\RomanHoI4\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\AQI - Aquitaine.txt	2018-12-31 22:00:04.000000000 -0800
***************
*** 150,161 ****
--- 150,162 ----
  add_opinion_modifier = { target = TUN modifier = positive_0 }
  add_opinion_modifier = { target = UTA modifier = negative_10 }
  add_opinion_modifier = { target = YUG modifier = positive_0 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_aquitaine
  }
  set_stability = 0.60
