*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\AST - Australia.txt	2019-01-10 19:00:52.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\AST - Australia.txt	2019-01-10 19:00:49.000000000 -0800
***************
*** 52,64 ****
          }
          neutrality = {
              popularity = 84
          }
      }
      
!     ruling_party = absolutist
      last_election = "1836.1.1"
      election_frequency = 48
      elections_allowed = no
  }
  add_opinion_modifier = { target = BEN modifier = positive_5 }
  add_opinion_modifier = { target = BIR modifier = negative_15 }
--- 52,64 ----
          }
          neutrality = {
              popularity = 84
          }
      }
      
!     ruling_party = neutrality
      last_election = "1836.1.1"
      election_frequency = 48
      elections_allowed = no
  }
  add_opinion_modifier = { target = BEN modifier = positive_5 }
  add_opinion_modifier = { target = BIR modifier = negative_15 }
***************
*** 71,95 ****
  add_opinion_modifier = { target = SHN modifier = negative_10 }
  add_opinion_modifier = { target = SOK modifier = positive_5 }
  add_opinion_modifier = { target = UTA modifier = negative_15 }
  
  
  add_ideas = {
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
  set_war_support = 0.60
  create_country_leader = {
      name = "Adamantios Dousmanis"
      desc = "POLITICS_ADAMANTIOS_DOUSMANIS_DESC"
      picture = "gfx/leaders/OC/portrait_BYZ_John_VIII.dds"
      expire = "1965.1.1"
!     ideology = absolute_monarchy
      traits = {
      }
  }
  create_corps_commander = {
  	name = "Themistoklis Kondylis"
  	picture = ""
--- 71,96 ----
  add_opinion_modifier = { target = SHN modifier = negative_10 }
  add_opinion_modifier = { target = SOK modifier = positive_5 }
  add_opinion_modifier = { target = UTA modifier = negative_15 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
  set_war_support = 0.60
  create_country_leader = {
      name = "Adamantios Dousmanis"
      desc = "POLITICS_ADAMANTIOS_DOUSMANIS_DESC"
      picture = "gfx/leaders/OC/portrait_BYZ_John_VIII.dds"
      expire = "1965.1.1"
!     ideology = minarchism_neutral
      traits = {
      }
  }
  create_corps_commander = {
  	name = "Themistoklis Kondylis"
  	picture = ""
