*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\AZR - Azerbaijan.txt	2019-01-10 16:45:36.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\AZR - Azerbaijan.txt	2019-01-10 16:40:00.000000000 -0800
***************
*** 74,85 ****
--- 74,86 ----
  add_opinion_modifier = { target = SHN modifier = negative_10 }
  add_opinion_modifier = { target = SOK modifier = negative_10 }
  add_opinion_modifier = { target = UTA modifier = negative_20 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
