*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\BRI - Brittany.txt	2019-01-07 18:52:29.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\BRI - Brittany.txt	2019-01-08 20:42:45.000000000 -0800
***************
*** 75,86 ****
--- 75,87 ----
  add_opinion_modifier = { target = SHN modifier = negative_30 }
  add_opinion_modifier = { target = SOK modifier = negative_30 }
  add_opinion_modifier = { target = UTA modifier = negative_10 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_breton
  }
  set_stability = 0.60
***************
*** 598,609 ****
  	defense_skill = 1
  	planning_skill = 1
  	logistics_skill = 1
  }
  
  create_navy_leader = {
! 	name = "Konwal Bréhat"
  	picture = ""
  	traits = { }
  	skill = 1
  }
  
--- 599,610 ----
  	defense_skill = 1
  	planning_skill = 1
  	logistics_skill = 1
  }
  
  create_navy_leader = {
! 	name = "Konwal Le Bras"
  	picture = ""
  	traits = { }
  	skill = 1
  }
  
