*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\BRG - Burgundy.txt	2019-01-10 16:59:01.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\BRG - Burgundy.txt	2019-01-10 16:59:03.000000000 -0800
***************
*** 86,97 ****
--- 86,98 ----
  add_opinion_modifier = { target = SHN modifier = negative_30 }
  add_opinion_modifier = { target = SOK modifier = negative_50 }
  add_opinion_modifier = { target = UTA modifier = negative_10 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_burgundian
  }
  set_stability = 0.60
