*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\BRK - Burkina Faso.txt	2019-01-01 08:05:04.000000000 -0800
--- D:\My Projects\RomanHoI4\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\BRK - Burkina Faso.txt	2019-01-01 08:05:07.000000000 -0800
***************
*** 155,166 ****
--- 155,167 ----
  add_opinion_modifier = { target = TUN modifier = positive_0 }
  add_opinion_modifier = { target = UTA modifier = negative_15 }
  add_opinion_modifier = { target = YUG modifier = positive_0 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_mossi
  }
  set_stability = 0.60
