*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\BRK - Burkina Faso.txt	2019-01-10 16:59:43.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\BRK - Burkina Faso.txt	2019-01-10 16:59:44.000000000 -0800
***************
*** 73,84 ****
--- 73,85 ----
  add_opinion_modifier = { target = MUT modifier = positive_15 }
  add_opinion_modifier = { target = SOK modifier = positive_15 }
  add_opinion_modifier = { target = UTA modifier = negative_15 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_mossi
  }
  set_stability = 0.60
