*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\CMR - Cameroon.txt	2019-01-08 21:03:41.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\CMR - Cameroon.txt	2019-01-08 21:07:23.000000000 -0800
***************
*** 80,91 ****
--- 80,92 ----
  add_opinion_modifier = { target = MUT modifier = positive_15 }
  add_opinion_modifier = { target = SOK modifier = positive_15 }
  add_opinion_modifier = { target = UTA modifier = negative_15 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_fang
  }
  set_stability = 0.60
