*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\CAT - Catalonia.txt	2019-01-10 19:06:26.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\CAT - Catalonia.txt	2019-01-10 19:06:22.000000000 -0800
***************
*** 71,82 ****
--- 71,83 ----
  add_opinion_modifier = { target = MUT modifier = positive_15 }
  add_opinion_modifier = { target = SOK modifier = positive_15 }
  add_opinion_modifier = { target = UTA modifier = negative_15 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_andalucian
  }
  set_stability = 0.60
