*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\CRO - Croatia.txt	2019-01-10 19:25:12.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\CRO - Pannonia.txt	2019-01-10 19:17:51.000000000 -0800
***************
*** 45,73 ****
  
  
  set_politics = {
  
      parties = {
          absolutist = {
!             popularity = 16
          }
          communism = {
!             popularity = 14
          }
          democratic = {
!             popularity = 41
          }
          fascism = {
!             popularity = 23
          }
          neutrality = {
!             popularity = 6
          }
      }
      
!     ruling_party = absolutist
      last_election = "1836.1.1"
      election_frequency = 48
      elections_allowed = no
  }
  add_opinion_modifier = { target = ADA modifier = positive_5 }
  add_opinion_modifier = { target = BEN modifier = positive_15 }
--- 45,73 ----
  
  
  set_politics = {
  
      parties = {
          absolutist = {
!             popularity = 6
          }
          communism = {
!             popularity = 11
          }
          democratic = {
!             popularity = 24
          }
          fascism = {
!             popularity = 3
          }
          neutrality = {
!             popularity = 56
          }
      }
      
!     ruling_party = neutrality
      last_election = "1836.1.1"
      election_frequency = 48
      elections_allowed = no
  }
  add_opinion_modifier = { target = ADA modifier = positive_5 }
  add_opinion_modifier = { target = BEN modifier = positive_15 }
***************
*** 79,100 ****
  add_opinion_modifier = { target = MUT modifier = positive_15 }
  add_opinion_modifier = { target = SOK modifier = positive_15 }
  add_opinion_modifier = { target = UTA modifier = negative_15 }
  
  
  add_ideas = {
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
  set_war_support = 0.60
  create_country_leader = {
      name = "Charilaos Metaxas"
      desc = "POLITICS_CHARILAOS_METAXAS_DESC"
      picture = "gfx/leaders/OC/portrait_BYZ_John_VIII.dds"
      expire = "1965.1.1"
!     ideology = absolute_monarchy
      traits = {
      }
  }
--- 79,101 ----
  add_opinion_modifier = { target = MUT modifier = positive_15 }
  add_opinion_modifier = { target = SOK modifier = positive_15 }
  add_opinion_modifier = { target = UTA modifier = negative_15 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
  set_war_support = 0.60
  create_country_leader = {
      name = "Charilaos Metaxas"
      desc = "POLITICS_CHARILAOS_METAXAS_DESC"
      picture = "gfx/leaders/OC/portrait_BYZ_John_VIII.dds"
      expire = "1965.1.1"
!     ideology = absolute_monarchy_neutral
      traits = {
      }
  }
