*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\FLD - Flanders-Wallonia.txt	2019-01-10 17:15:36.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\FLD - Flanders-Wallonia.txt	2019-01-10 17:10:49.000000000 -0800
***************
*** 83,94 ****
--- 83,95 ----
  add_opinion_modifier = { target = SHN modifier = negative_10 }
  add_opinion_modifier = { target = SOK modifier = positive_5 }
  add_opinion_modifier = { target = UTA modifier = negative_55 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_wallonian
  }
  set_stability = 0.60
