*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\FRA - France.txt	2019-01-10 17:16:32.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\FRA - France.txt	2019-01-10 17:16:34.000000000 -0800
***************
*** 48,66 ****
              popularity = 6
          }
          communism = {
              popularity = 17
          }
          democratic = {
!             popularity = 44
          }
          fascism = {
              popularity = 23
          }
          neutrality = {
!             popularity = 10
          }
      }
      
      ruling_party = neutrality
      last_election = "1918.3.22"
      election_frequency = 48
--- 48,66 ----
              popularity = 6
          }
          communism = {
              popularity = 17
          }
          democratic = {
!             popularity = 10
          }
          fascism = {
              popularity = 23
          }
          neutrality = {
!             popularity = 42
          }
      }
      
      ruling_party = neutrality
      last_election = "1918.3.22"
      election_frequency = 48
***************
*** 84,95 ****
--- 84,96 ----
  add_opinion_modifier = { target = SHN modifier = positive_5 }
  add_opinion_modifier = { target = SOK modifier = positive_5 }
  add_opinion_modifier = { target = UTA modifier = positive_5 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_cosmopolitan_french
  }
  set_stability = 0.60
