*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\GAB - Gabon.txt	2019-01-07 18:52:29.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\GAB - Gabon.txt	2019-01-08 20:48:16.000000000 -0800
***************
*** 78,89 ****
--- 78,90 ----
  add_opinion_modifier = { target = COG modifier = positive_106 }
  add_opinion_modifier = { target = ETH modifier = positive_105 }
  add_opinion_modifier = { target = SOK modifier = positive_75 }
  
  
  add_ideas = {
+ 	RME_former_dominion
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_fang
  }
  set_stability = 0.60
