*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\GEO - Georgia.txt	2019-01-10 17:18:12.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\GEO - Georgia.txt	2019-01-10 17:18:14.000000000 -0800
***************
*** 79,90 ****
--- 79,91 ----
  add_opinion_modifier = { target = KHI modifier = positive_30 }
  add_opinion_modifier = { target = MUT modifier = positive_30 }
  add_opinion_modifier = { target = SOK modifier = positive_30 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
