*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\GRB - Great Britain.txt	2019-01-01 10:42:03.000000000 -0800
--- D:\My Projects\RomanHoI4\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\GRB - Great Britain.txt	2019-01-01 10:45:24.000000000 -0800
***************
*** 161,172 ****
--- 161,173 ----
  add_opinion_modifier = { target = TUN modifier = positive_0 }
  add_opinion_modifier = { target = UTA modifier = negative_100 }
  add_opinion_modifier = { target = YUG modifier = positive_0 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	great_power
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_english
  }
