*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\GRB - Great Britain.txt	2019-01-07 18:52:29.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\GRB - Great Britain.txt	2019-01-08 20:49:08.000000000 -0800
***************
*** 110,121 ****
--- 110,122 ----
  add_opinion_modifier = { target = SUK modifier = positive_5 }
  add_opinion_modifier = { target = TIB modifier = negative_50 }
  add_opinion_modifier = { target = UTA modifier = negative_100 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	great_power
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_english
  }
