*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\HUN - Hungary.txt	2019-01-10 17:22:55.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\HUN - Hungary.txt	2019-01-10 17:22:55.000000000 -0800
***************
*** 84,99 ****
  add_opinion_modifier = { target = SHN modifier = negative_93 }
  add_opinion_modifier = { target = SOK modifier = positive_45 }
  add_opinion_modifier = { target = UTA modifier = positive_121 }
  
  
  add_ideas = {
! 	volunteer_only
! 	civilian_economy
  	export_focus
  	culture_hungarian
  }
  set_stability = 0.60
  set_war_support = 0.61
  create_country_leader = {
      name = "Ottó Széchenyi"
      desc = "POLITICS_OTTó_SZéCHENYI_DESC"
--- 84,100 ----
  add_opinion_modifier = { target = SHN modifier = negative_93 }
  add_opinion_modifier = { target = SOK modifier = positive_45 }
  add_opinion_modifier = { target = UTA modifier = positive_121 }
  
  
  add_ideas = {
! 	disarmed_nation
! 	isolation
  	export_focus
  	culture_hungarian
+ 	HUN_bavarian_demostrations
  }
  set_stability = 0.60
  set_war_support = 0.61
  create_country_leader = {
      name = "Ottó Széchenyi"
      desc = "POLITICS_OTTó_SZéCHENYI_DESC"
