*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\ISR - Israel.txt	2019-01-01 10:15:05.000000000 -0800
--- D:\My Projects\RomanHoI4\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\ISR - Israel.txt	2019-01-01 10:15:50.000000000 -0800
***************
*** 154,165 ****
--- 154,166 ----
  add_opinion_modifier = { target = TUN modifier = positive_0 }
  add_opinion_modifier = { target = UTA modifier = negative_20 }
  add_opinion_modifier = { target = YUG modifier = positive_0 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_ashkenazi
  }
  set_stability = 0.60
