*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\ISR - Israel.txt	2019-01-07 18:52:29.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\ISR - Israel.txt	2019-01-08 20:50:30.000000000 -0800
***************
*** 77,88 ****
--- 77,89 ----
  add_opinion_modifier = { target = SHN modifier = negative_10 }
  add_opinion_modifier = { target = SOK modifier = positive_20 }
  add_opinion_modifier = { target = UTA modifier = negative_20 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_ashkenazi
  }
  set_stability = 0.60
