*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\JAV - Java.txt	2019-01-07 18:52:29.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\JAV - Java.txt	2019-01-08 20:51:19.000000000 -0800
***************
*** 75,86 ****
--- 75,87 ----
  add_opinion_modifier = { target = SHN modifier = positive_5 }
  add_opinion_modifier = { target = SOK modifier = negative_20 }
  add_opinion_modifier = { target = UTA modifier = negative_10 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
