*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\MEX - Mexico.txt	2019-01-10 17:47:46.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\MEX - Mexico.txt	2019-01-10 17:47:46.000000000 -0800
***************
*** 72,84 ****
  add_opinion_modifier = { target = SOK modifier = positive_11 }
  add_opinion_modifier = { target = UTA modifier = positive_107 }
  
  
  add_ideas = {
  	volunteer_only
! 	civilian_economy
  	export_focus
  	culture_nahua
  }
  set_stability = 0.60
  set_war_support = 0.60
  create_country_leader = {
--- 72,84 ----
  add_opinion_modifier = { target = SOK modifier = positive_11 }
  add_opinion_modifier = { target = UTA modifier = positive_107 }
  
  
  add_ideas = {
  	volunteer_only
! 	low_economic_mobilisation
  	export_focus
  	culture_nahua
  }
  set_stability = 0.60
  set_war_support = 0.60
  create_country_leader = {
