*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\MOR - Morocco.txt	2019-01-10 17:56:35.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\MOR - Morocco.txt	2019-01-10 17:51:13.000000000 -0800
***************
*** 48,63 ****
  
      parties = {
          absolutist = {
              popularity = 16
          }
          communism = {
!             popularity = 14
          }
          democratic = {
!             popularity = 41
          }
          fascism = {
              popularity = 23
          }
          neutrality = {
              popularity = 6
--- 48,63 ----
  
      parties = {
          absolutist = {
              popularity = 16
          }
          communism = {
!             popularity = 21
          }
          democratic = {
!             popularity = 34
          }
          fascism = {
              popularity = 23
          }
          neutrality = {
              popularity = 6
***************
*** 80,91 ****
--- 80,92 ----
  add_opinion_modifier = { target = SHN modifier = negative_20 }
  add_opinion_modifier = { target = SOK modifier = negative_20 }
  add_opinion_modifier = { target = UTA modifier = negative_10 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
