*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\MOR - Morocco.txt	2019-01-01 10:42:04.000000000 -0800
--- D:\My Projects\RomanHoI4\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\MOR - Morocco.txt	2019-01-01 10:44:38.000000000 -0800
***************
*** 136,147 ****
--- 136,148 ----
  add_opinion_modifier = { target = TUN modifier = positive_0 }
  add_opinion_modifier = { target = UTA modifier = negative_10 }
  add_opinion_modifier = { target = YUG modifier = positive_0 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
