*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\NMB - Namibia.txt	2019-01-07 18:52:29.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\NMB - Namibia.txt	2019-01-08 20:53:57.000000000 -0800
***************
*** 80,91 ****
--- 80,92 ----
  add_opinion_modifier = { target = MUT modifier = positive_106 }
  add_opinion_modifier = { target = RME modifier = positive_197 }
  add_opinion_modifier = { target = SAF modifier = positive_105 }
  
  
  add_ideas = {
+ 	RME_former_dominion
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
