*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\RME - The Roman Empire.txt	2019-01-01 11:09:56.000000000 -0800
--- D:\My Projects\RomanHoI4\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\RME - The Roman Empire.txt	2019-01-01 10:16:48.000000000 -0800
***************
*** 55,70 ****
  
      parties = {
          absolutist = {
              popularity = 16
          }
          communism = {
!             popularity = 1
          }
          democratic = {
!             popularity = 54
          }
          fascism = {
              popularity = 23
          }
          neutrality = {
              popularity = 6
--- 55,70 ----
  
      parties = {
          absolutist = {
              popularity = 16
          }
          communism = {
!             popularity = 14
          }
          democratic = {
!             popularity = 41
          }
          fascism = {
              popularity = 23
          }
          neutrality = {
              popularity = 6
***************
*** 168,188 ****
  add_ideas = {
  	great_power
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
  set_war_support = 0.68
  create_country_leader = {
!     name = "Andreas Staikopoulos"
!     desc = "POLITICS_ANDREAS_STAIKOPOULOS_DESC"
!     picture = "gfx/leaders/OC/portrait_BYZ_John_VIII.dds"
      expire = "1965.1.1"
      ideology = absolute_monarchy
      traits = {
      }
  }
  create_corps_commander = {
  	name = "Diomidis Venizelos"
  	picture = ""
--- 168,199 ----
  add_ideas = {
  	great_power
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
+ 	michael_VII
  }
  set_stability = 0.60
  set_war_support = 0.68
  create_country_leader = {
!     name = "Konstantinos Doukas"
!     desc = "POLITICS_KONSTANTINOS_DOUKAS_DESC"
!     picture = "gfx/leaders/ENG/Portrait_Britain_Edward_VIII.dds"
      expire = "1965.1.1"
      ideology = absolute_monarchy
+     traits = {
+ 		inexperienced_imperialist
+     }
+ }
+ create_country_leader = {
+     name = "Tiberius Julius Aurelius"
+     desc = "POLITICS_KONSTANTINOS_DOUKAS_DESC"
+     picture = "gfx/leaders/Europe/Portrait_Europe_Generic_3.dds"
+     expire = "1965.1.1"
+     ideology = fascism_ideology
      traits = {
      }
  }
  create_corps_commander = {
  	name = "Diomidis Venizelos"
  	picture = ""
