*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\RUS - Russia.txt	2019-01-10 18:07:53.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\RUS - Russia.txt	2019-01-10 18:07:57.000000000 -0800
***************
*** 108,123 ****
  add_opinion_modifier = { target = SOK modifier = negative_200 }
  add_opinion_modifier = { target = UTA modifier = negative_200 }
  
  
  add_ideas = {
  	great_power
! 	volunteer_only
! 	civilian_economy
  	export_focus
  	culture_russian
  }
  set_stability = 0.60
  set_war_support = 0.61
  create_country_leader = {
      name = "Maxim Kuropatkin"
      desc = "POLITICS_MAXIM_KUROPATKIN_DESC"
--- 108,125 ----
  add_opinion_modifier = { target = SOK modifier = negative_200 }
  add_opinion_modifier = { target = UTA modifier = negative_200 }
  
  
  add_ideas = {
  	great_power
! 	disarmed_nation
! 	isolation
  	export_focus
  	culture_russian
+ 	RUS_ukrainian_separatism_1
+ 	RUS_ENG_alliance
  }
  set_stability = 0.60
  set_war_support = 0.61
  create_country_leader = {
      name = "Maxim Kuropatkin"
      desc = "POLITICS_MAXIM_KUROPATKIN_DESC"
