*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\SAF - South Africa.txt	2019-01-07 18:52:30.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\SAF - South Africa.txt	2019-01-08 20:57:20.000000000 -0800
***************
*** 83,94 ****
--- 83,95 ----
  add_opinion_modifier = { target = SHN modifier = negative_10 }
  add_opinion_modifier = { target = SOK modifier = positive_5 }
  add_opinion_modifier = { target = UTA modifier = negative_15 }
  
  
  add_ideas = {
+ 	RME_former_dominion
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
