*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\TRI - Tripoli.txt	2019-01-07 18:52:30.000000000 -0800
--- D:\My Projects\RomanHoI4_cv_changes\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\TRI - Tripoli.txt	2019-01-08 21:00:41.000000000 -0800
***************
*** 87,98 ****
--- 87,99 ----
  add_opinion_modifier = { target = SOK modifier = negative_20 }
  add_opinion_modifier = { target = SYR modifier = positive_15 }
  add_opinion_modifier = { target = UTA modifier = negative_10 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
