*** D:\My Projects\RomanHoI4\results\The_Roman_Empire1936_01_01\history\countries\TUN - Tunis.txt	2019-01-01 11:09:57.000000000 -0800
--- D:\My Projects\RomanHoI4\HoI4Mod\The_Roman_Empire1936_01_01\history\countries\TUN - Tunis.txt	2019-01-01 11:13:38.000000000 -0800
***************
*** 161,172 ****
--- 161,173 ----
  add_opinion_modifier = { target = TRI modifier = positive_0 }
  add_opinion_modifier = { target = UTA modifier = negative_10 }
  add_opinion_modifier = { target = YUG modifier = positive_0 }
  
  
  add_ideas = {
+ 	RME_former_territory
  	volunteer_only
  	civilian_economy
  	export_focus
  	culture_greek
  }
  set_stability = 0.60
